geode-assembly/src/acceptanceTest/resources/gradle-test-projects/management/build.gradle

```diff
@@ -26,7 +26,7 @@ repositories {
 }
 
 dependencies {
-    compile("org.apache.geode:geode-core:${findProperty('geodeVersion')}")
+    compile("${findProperty('projectGroup')}:geode-core:${findProperty('geodeVersion')}")
```

Comment:
See top-level comment, but this should probably read
`compile("${project.group}:geode-core:${project.version}")`